Makefile

```diff
@@ -41,6 +42,9 @@ tests:
 	@rm -fr .coverage
 	$(VENV)/bin/tox
 
+functional: need-kinto-running
```

Comment:
I named it like that because it is how we did elsewhere e.g [kinto-signer](https://github.com/Kinto/kinto-signer/blob/master/Makefile#L62-L63) and [kinto-emailer](https://github.com/Kinto/kinto-emailer/blob/add-tests/Makefile#L53-L54)